packages/@aws-cdk/aws-dynamodb/README.md

```diff
@@ -73,6 +73,33 @@ Further reading:
 https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/AutoScaling.html
 https://aws.amazon.com/blogs/database/how-to-use-aws-cloudformation-to-configure-auto-scaling-for-amazon-dynamodb-tables-and-indexes/
 
+### Encryption
+
+All user data stored in Amazon DynamoDB is fully encrypted at rest. When creating a new table, you can choose to encrypt using the following customer master keys (CMK) to encrypt your table:
+* AWS owned CMK – Default encryption type. The key is owned by DynamoDB (no additional charge).
+* AWS managed CMK – The key is stored in your account and is managed by AWS KMS (AWS KMS charges apply).
+* Customer managed CMK – The key is stored in your account and is created, owned, and managed by you. You have full control over the CMK (AWS KMS charges apply).
+
+You may choose to use an AWS Managed CMK in your account by enabling `serverSideEncryption`
+
+```ts
+import dynamodb = require('@aws-cdk/aws-dynamodb');
+
+const table = new dynamodb.Table(this, 'Table', {
+  serverSideEncryption: true
+});
+```
+
+To specify a Customer managed CMK, pass the CMK's key ID, Amazon Resource Name (ARN), alias name, or alias ARN to `kmsMasterKeyId`
+
+```ts
+import dynamodb = require('@aws-cdk/aws-dynamodb');
+
+const table = new dynamodb.Table(this, 'Table', {
+  kmsMasterKeyId: 'cmkKeyID'
```

Comment:
It would be better if this accepted a `kmsMasterKey` that is a `kms.IKey`. This would also allow granting DynamoDB permissions on the key as is necessary.

Reply:
Added passing of kms.IKey via constructor. 

Also I added the creation of an IAM Policy the user will use to let a principal access KMS via DynamoDB. The steps for this are spelled out here, [https://docs.aws.amazon.com/kms/latest/developerguide/services-dynamodb.html#dynamodb-customer-cmk-policy](https://docs.aws.amazon.com/kms/latest/developerguide/services-dynamodb.html#dynamodb-customer-cmk-policy)